### DISCUSSION TITLE
is there any front end example?

### DISCUSSION THREAD

**Discussion by kouohhashi** [payloadcms/payload#659] [Q&A] [answered]
Hi, I'm new here.

I understand Payload is a headless CMS so I need to develop front end web sites to receive data from Payload and show visual representation. And my question is is there any examples for scaffold front end projects which I can learn how to connect data from Payload and front end.

Thanks in advance.

**Comment by jmikrut** [comment, marked as answer]
Hey @kouohhashi — welcome!

We do have an in-depth YouTube series that shows how to create an entire website, including Payload and NextJS, from scratch. 

You can find the first video in the series here:
https://www.youtube.com/watch?v=bxWsZTtqs80

This video in specific combines Next and Payload into a single server, which is not necessarily applicable for huge projects, but can offer some simplicity overall. Lots of stuff can be learned.

Here's the code:
https://github.com/payloadcms/custom-website-series

We're planning on releasing lots more of these soon too!

**Reply by kouohhashi** [reply]
Thanks I'll try it!
